doc/nrf/releases/release-notes-latest.rst

```diff
@@ -187,6 +294,23 @@ The following list summarizes the most important changes inherited from upstream
     * Unified endpoint helper macros across all USB device drivers.
     * Fixed handling of fragmented transfers on the control OUT endpoint in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Introduced names for threads used in USB classes, to aid debugging.
+    * Updated the way the :cpp:func:`usb_enable` function should be used.
+      For some samples, this function was invoked automatically on system boot-up to enable the USB subsystem, but now it must be called explicitly by the application.
+      If your application relies on any of the following Kconfig options, it must also enable the USB subsystem:
+
+      * :option:`CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_ECM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_EEM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_RNDIS`
+      * :option:`CONFIG_TRACING_BACKEND_USB`
+      * :option:`CONFIG_USB_UART_CONSOLE`
+
+    * Updated the implementation of the CDC class to not duplicate USB transfer slots for OUT transfers.
+    * Updated the implementation of CDC ACM to re-trigger transfer for the OUT endpoint on resume.
+    * Fixed an issue with remote wake-up requests in the nRF driver.
+    * Updated the implementation of the HID class to allow sending data only in CONFIGURED state.
+    * Switched to using the kernel stack.
```

Comment:
This one says nothing to me and remind me nothing. Could You please point me to the commit that You base this entry on?

Reply:
As you wish: https://github.com/zephyrproject-rtos/zephyr/pull/27308

Reply:
``` diff
-     * Switched to using the kernel stack.
+     * Use kernel stack for threads not running in user space.
```

Reply:
```suggestion
    * Updated to use the kernel stack for thread not running in user space.
```